e2e/nomostest/helm/helm.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"kpt.dev/configsync/e2e/nomostest"
+	nomostesting "kpt.dev/configsync/e2e/nomostest/testing"
+	"kpt.dev/configsync/e2e/nomostest/testshell"
+	"sigs.k8s.io/kustomize/kyaml/copyutil"
+)
+
+// PrivateARHelmRegistry is the registry URL to the private AR used for testing
+var PrivateARHelmRegistry = fmt.Sprintf("oci://us-docker.pkg.dev/%s/config-sync-test-ar-helm", nomostesting.GCPProjectIDFromEnv)
+
+// PrivateARHelmHost is the host name of the private AR used for testing
+var PrivateARHelmHost = "https://us-docker.pkg.dev"
+
+// RemoteHelmChart represents a remote OCI-based helm chart
+type RemoteHelmChart struct {
+	// Shell is a helper utility to execute shell commands in a test.
+	Shell *testshell.TestShell
+
+	// Host is the host URL, e.g. https://us-docker.pkg.dev
+	Host string
+
+	// Registry is the registry URL, e.g. oci://us-docker.pkg.dev/stolos-dev/config-sync-test-ar-helm
```

Comment:
`stolos-dev` is legacy at this point, suggest using a different example/placeholder

Reply:
Done!